metrics/prometheus.go

```diff
@@ -500,6 +500,11 @@ func (c *PrometheusCollector) Collect(ch chan<- prometheus.Metric) {
 	c.errors.Collect(ch)
 }
 
+const (
+	dockerLabelPrefix = "docker_label_"
+	dockerEnvPrefix   = "docker_env_"
```

Comment:
How about just `container_{label,env}_`? At least that will be consistent for dashboards, etc across docker/rkt envs.